src/Page/Definition.elm

```diff
@@ -84,23 +86,38 @@ renderExpandableCategories model categories =
         (\listPosition ->
             -- We maybe want to ditch this div for valid html - but this is simple
             div []
-                [ renderTerm (categoryKeysFromListPosition listPosition)
+                [ renderTerm model (categoryKeysFromListPosition listPosition)
                 , renderDefinition model (categoryKeysFromListPosition listPosition)
                 ]
         )
         categories
 
 
-renderTerm : CategoryDefinition -> Html Msg
-renderTerm category =
-    dt [] [ button [ onClick (ToggleCategory category.title) ] [ text (t category.title) ] ]
+renderTerm : Model -> CategoryDefinition -> Html Msg
+renderTerm model category =
+    if isExpanded model category.title then
+        dt [ css [ expanderItemStyle ] ]
+            [ button [ onClick (ToggleCategory category.title), css [ expanderButtonStyle ] ]
+                [ h2 [ css [ expanderHeadingStyle ] ] [ text (t category.title) ]
+                , span [ css [ expanderSymbolStyle, rotate90Style ] ] [ text ">" ]
```

Comment:
Is the rotate style the only difference here? If so, we could make only that conditional.